src/reconcile/rec_visibility.c

```diff
@@ -350,11 +350,18 @@ __wt_rec_upd_select(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_INSERT *ins, v
             upd_select->upd = upd = upd->next;
         }
         if (upd != NULL) {
-            /* The beginning of the validity window is the selected update's time pair. */
-            if (upd->start_ts < upd_select->stop_ts)
+            /*
+             * The beginning of the validity window is the selected update's time pair.
+             *
+             * FIXME-PM-1521: We shouldn't need to check for this. We're actually allowed to commit
```

Comment:
Updates can be committed out of timestamp order. So we can have updates with an earlier stop time pair than their start time pair. This breaks a lot of assumptions we've made so there is going to be a discussion with the execution team about whether we need to continue supporting this. For now, I'm just guarding this check to prevent failures.